### ISSUE TITLE
Usage des variables host et port de install_db

### ISSUE THREAD

**Issue by lpofredc** [PnX-SI/GeoNature-atlas#229]
Les variables `db_host` & 'db_port' sont utilisées de manière relativement aléatoires dans le script install_db.sh ce qui, selon les cas de figure, peut rendre l'installation complexe:

- Parfois connexion directe via socket unix (donc sans hote ni port);
- Parfois avec l'hote uniquement;
- Parfois avec le port uniquement.

Je vais voir pour homogénéiser tout ça.

**Comment by gildeluermoz** [comment]
Yes, j'ai vu que notamment on utilise pas la variable concernant le port.
Par contre on est en train de tout revoir le install_db.sh avec Kévin
https://github.com/PnX-SI/GeoNature/commits/deb_compat
Donc idéalement, il faudrait faire ce travail après nos modifs ou que tu me précises ce que tu as identifié pour que j'en profite de l'harmoniser moi même au cours de ce travail.

**Comment by camillemonchicourt** [comment]
Oui là ici ça concerne GN-atlas. 
Mais dans GN il y a des problèmes similaires et un utilisateur a d'ailleurs forké le dépôt GN récemment et amélioré différentes choses, dont celle-ci des ports et hosts : https://github.com/lolofr/GeoNature/commit/f0ed145c5893aec9086100929a026c380dcca1b7

**Comment by lpofredc** [comment]
Oui, merci.
il faut toutefois être prudent car, par défaut, dès que l'on fait appel à psql en spécifiant un hôte, un change de mode d'authentification (généralement md5). il est ainsi demandé un mot de passe même pour le compte postgres (qui n'en a pas par défaut...)

**Comment by gildeluermoz** [comment]
Cool, Merci. Ca me fait une base pour le faire. J'avais ça dans ma todo liste. 


	
	Gil DELUERMOZ 
Chargé de mission développement web et bases de données 

Parc national des Écrins 
Tél : +33 (0) 4 92 40 20 38 (lundi) 
+33 (0)4 92 21 08 49 (mardi, mercredi matin et jeudi) 
[ http://www.ecrins-parcnational.fr/ | www.ecrins-parcnational.fr ] 


[ http://actu.ecrins.info/ ] 


De: "Fred C" <notifications@github.com> 
À: "PnX-SI/GeoNature-atlas" <GeoNature-atlas@noreply.github.com> 
Cc: "Gil DELUERMOZ" <gil.deluermoz@ecrins-parcnational.fr>, "Comment" <comment@noreply.github.com> 
Envoyé: Jeudi 12 Mars 2020 00:18:20 
Objet: Re: [PnX-SI/GeoNature-atlas] Usage des variables host et port de install_db (#229) 





Oui, merci. 
il faut toutefois être prudent car, par défaut, dès que l'on fait appel à psql en spécifiant un hôte, un change de mode d'authentification (généralement md5). il est ainsi demandé un mot de passe même pour le compte postgres (qui n'en a pas par défaut...)

**Comment by lpofredc** [comment]
cf. PR #234

**Comment by lpofredc** [comment]
L'installation "docker" fait le job
